glib/src/auto/functions.rs

```diff
@@ -695,6 +772,13 @@ pub fn shell_unquote(
     }
 }
 
+#[doc(alias = "g_slice_debug_tree_statistics")]
+pub fn slice_debug_tree_statistics() {
```

Comment:
This one is only available for debug builds, we should probably drop it? or add a debug feature?